index.html

```diff
@@ -0,0 +1,34 @@
+<!-- Boilerplate 1-10-->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie-edge">
+    <meta name="viewport" content="width=device-width" , initial-scale="1.0">
+    <title>Language Switcher</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<!-- Main-section 14-31-->
+
+<body>
+    <div class="container">
+        <div class="language-section">
+            <button class="button active" data-language="english">English</button>
+            <button class="button" data-language="french">French</button>
+            <button class="button" data-language="german">German</button>
+        </div>
+        <div class="content">
+            <h1 class="title">
+                Translating static text.
+            </h1>
+            <p class="description">
+                Translating static text.
+            </p>
+        </div>
+
+    </div>
+</body>
+
+</html>
```

Comment:
Git *reallllly* wants all files to end in an empty return.

I use a tool to ensure this happens automatically.

In the root of this repo create a `.editorconfig` file with this contents:

```yml
# EditorConfig is awesome: http://EditorConfig.org

# Top-most EditorConfig file
root = true

# defaults for all files
[*]
charset = utf-8
end_of_line = lf
indent_size = 2
indent_style = space
insert_final_newline = true
trim_trailing_whitespace = true

# Markdown files uses two trailing spaces to indicate a <br>
[*.{md,snap}]
trim_trailing_whitespace = false

# Sass files are indent based, 4 spaces makes them easier to read
[*.{sass,scss,css}]
indent_size = 4
```
Then in your editor look up EditorConfig plugin and install it.

From now on, any time you hit save on a file in this repo it will make sure there is at least one empty return at the end of the file, and if not it will add it.

Reply:
Thanks for the suggestion, I added this.